CONTRIBUTING.md

```diff
@@ -160,6 +160,28 @@ Lint the project with:
     # for javascript
     npm run lint
 
+## Linting with codeclimate
```

Comment:
I'd had a note:

codeclimate is a service we use to measure code quality and test coverage. To get codeclimate's report on your branch, ideally before sending your PR, you can setup codeclimate against your Caravel fork. After you push to your fork, you should be able to get the report at http://codeclimate.com . Alternatively, if you prefer to work locally, you can install the codeclimate cli tool.